res/type-defs-blockchain/common.graphql

```diff
@@ -8,3 +8,16 @@ type PageInfo {
     hasNextPage: Boolean!
     hasPreviousPage: Boolean!
 }
+
+enum AddressFormat {
+    HEX
```

Comment:
remove ACCOUNT_ID, and rename HEX to RAW like here https://test.ton.org/HOWTO.txt